web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-troubleshooting.adoc

```diff
@@ -0,0 +1,166 @@
+= Troubleshoot Web Service Consumer Connector - Mule 4
+
+To troubleshoot Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), review the following sections for performing general troubleshooting, addressing specific common errors, and interpreting commonly thrown messages.
+
+== Enable Wire Logging
+
+When troubleshooting Web Service Consumer Connector it is helpful to look at the transport request and response on the Web Service Consumer.
```

Comment:
This is another of those modification errors: "it" is not doing the troubleshooting. More precisely:
```suggestion
To begin troubleshooting Web Service Consumer Connector, look at the transport request and response on the Web Service Consumer.
```